oer-bookr/src/styles/formStyles.js

```diff
@@ -32,5 +34,8 @@ export const Button = styled.button`
   ${props =>
     props.color === "primary"
       ? `background-color: #007bff; border-color: #007bff; color: #fff;`
-      : null}
+      : props.color === "secondary"
+      ? `background-color: #08a581; border-color: #08a581; color: #fff;`
+      : props.color === "danger" &&
```

Comment:
Since all that you use on the props object is color, I'd suggest destructuring color:
```javascript
${({ color }) => 
  color === "primary"
    ? // ...
```